t/recreate.t

```diff
@@ -3,7 +3,7 @@
 use Data::Dumper;
 use lib '.'; use lib 't';
 use SATest; sa_t_init("recreate");
-use Test;
+use Test::More tests => 9;
 
 BEGIN { 
```

Comment:
this BEGIN block seems to be a boilerplate, would consider an helper setup_inc from SATest (similar to the one used by the perl testsuite)